var/spack/repos/builtin/packages/thrift/package.py

```diff
@@ -30,12 +31,12 @@ class Thrift(Package):
     version("0.9.3", sha256="b0740a070ac09adde04d43e852ce4c320564a292f26521c46b78e0641564969e")
 
     # Currently only support for c-family and python
-    variant("c", default=True, description="Build support for C-family languages")
     variant("pic", default=True, description="Build position independent code")
+    variant("c", default=True, description="Build support for C-family languages")
     variant("python", default=True, description="Build support for python")
+    variant("java", default=False, description="Build support for java")
```

Comment:
If you're grouping variants by language, perhaps put the languages in alphabetical order?  I know it's a nit though.

Reply:
From https://thrift.apache.org/docs/install/ it looks like `java` needs to be at version `1.7`.  There's also a dependency on `ant` for `java`.